pages/2017/index.rst

```diff
@@ -8,4 +8,60 @@ Python in Astronomy 2017
 Lorentz Center, Leiden
 ----------------------
 
-WOOOO YEAG
+The third edition of the Python in Astronomy Conference will be held at the
+Lorentz Center on the **8 - 12th May 2017**.
+
+As with the two previous Python in Astronomy conferences we aim to bring a wide
+variety of people who use, develop or teach people about Python packages in the
+context of all forms of Astronomy. The conference will include presentations,
+tutorials unconference sessions and sprints. As well as building the community
+around astronomical uses of Python the conference aims to improve collaboration
+and interoperability between Python packages, share knowledge on Python packages
+and techniques and provide training and educational materials for users and
+developers of Python packages.
+
+Attending PyAstro17
+###################
+
+Between Sept 12, 2016 and Dec 9, 2016, we will be accepting requests from
```

Comment:
Not really my business, but since I'm reviewing this :laugh: - I think you should probably have an earlier deadline - it usually takes several weeks to make decisions, and it's best to give people as much time as possible to get plane tickets etc (ideally in the past we've aimed to send out decisions before the end of the year)